### ISSUE TITLE
Gemfile: Could you tell me why many gems don't have their specific version?

### ISSUE THREAD

**Issue by Jamal-CRF** [Jamal-CRF/lab_portal#2]

**Comment by ricardopacheco** [comment]
@Jamal-CRF Of course, I can. Rails is focused on MVP projects, and the idea is to leave everything as unlocked as possible. When you don't put specific versions, you give bundler a lot of flexibility, and, it will solve the absolute majority of problems (which is great on new projects and startup teams).

However, as the number of dependencies increases, and they are updated, the chance of conflicts in the bundle increases. When you "lock" the versions, you minimize these problems, having more control over the update.

It's not wrong for you to leave it loose, but in cases of very legacy apps, this is a bad practice.
